src/charts/SupersetEmbed/SupersetEmbed.js

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+import React, { useEffect, useRef, useState } from 'react';
+import PropTypes from 'prop-types';
+import { Box, CircularProgress, Backdrop } from '@mui/material';
+import { embedDashboard } from '@superset-ui/embedded-sdk';
+
+export const SupersetEmbed = ({ isLoading, guestToken, report, style, options }) => {
+  const containerRef = useRef(null);
+  const [isEmbedded, setIsEmbedded] = useState(false);
+
+  useEffect(() => {
+    const loadSuperset = async () => {
+      if (!guestToken || !report?.id || !options?.supersetUrl) return;
+
+      try {
+        setIsEmbedded(false);
+        await embedDashboard({
+          id: report.id,
+          supersetDomain: options.supersetUrl,
+          mountPoint: containerRef.current,
+          fetchGuestToken: async () => guestToken,
+          dashboardUiConfig: report?.uiConfig || {},
+        });
+        setIsEmbedded(true);
+      } catch (error) {
+        console.error('Superset embedding failed:', error);
+      }
+    };
+
+    loadSuperset();
+
+    if (containerRef.current && containerRef.current.children[0]) {
+      containerRef.current.children[0].style.width = '100%';
+      containerRef.current.children[0].style.height = '100%';
+    }
+  }, [guestToken, report, options]);
+
+  return (
+    <Box
+      sx={{
+        position: 'relative',
+        width: style?.width,
+        height: style?.height,
```

Comment:
Can we add fallback values here? (with the same values as the default values of the `style` prop):
```
        width: style?.width ?? '100%',
        height: style?.height ?? '800px',
```


The idea is that in the webapp, developers using this component may only define one of the two attributes in the `style` prop:
```
<SupersetEmbed style={{height:'123px'}} />
```

In this case, the value of `style.width` would be `undefined`, because the prop default value wouldn't be used.